src/event.c

```diff
@@ -123,23 +121,79 @@ static bool is_higher_than_threshold(knot_value_type value,
 
 static void on_sensor_to(struct l_timeout *to, void *data)
 {
-	struct data_item_timeout *data_item_to_info = data;
+	int *time_sec = data;
+
+	timeout_cb();
+	l_timeout_modify(to, *time_sec);
+}
+
+static int comp_change_lower_flag(knot_value_type current_val,
+				  knot_value_type sent_val,
+				  knot_value_type lower_limit,
+				  int value_type)
+{
+	int rc = KNOT_ERR_INVALID;
```

Comment:
What rc means?

Reply:
Return calculation. It is a common way of abbreviating.